src/components/Hearing/Section/SectionImage.js

```diff
@@ -3,12 +3,22 @@ import PropTypes from 'prop-types';
 import Lightbox from 'react-image-lightbox';
 
 export const SectionImageComponent = ({image, caption, title, showLightbox, openLightbox, closeLightbox}) => {
+  const defineImageAlt = () => {
+    let imageAlt = "";
+    if (caption) {
+      imageAlt = caption;
+    }
+    if (title) {
+      imageAlt = title;
+    }
+    return imageAlt;
```

Comment:
Shouldn't the caption take priority over title ? In general image alt terms ? because imageAlt in this case will almost always be title.

anywho. Perhaps this block can be a lot shorter without variable assignment.

```
const defineImageAlt = () => caption || title;
```

Reply:
In most of the current data, there is no title so the image caption is displayed as alt text.

However, in general, I believe caption to be always more descriptive than the image title, so I agree with @farrukh8925 that we should use the caption (or even better, alt text, if we want to add backend support for that).